telescope/selector.py

```diff
@@ -340,8 +340,8 @@ def _encode_multi_selector(self, selector):
         'start_times': self._encode_start_times(selector.start_times)
         }
 
-    if selector.site_names != [None]:
```

Comment:
Yeah, but our test was also broken (the test accidentally created this field) so we didn't catch it.